paper-server/src/main/java/io/papermc/paper/command/PaperPluginsCommand.java

```diff
@@ -36,35 +35,28 @@ public class PaperPluginsCommand extends BukkitCommand {
 
     private static final TextColor INFO_COLOR = TextColor.color(52, 159, 218);
 
-    // TODO: LINK?
     private static final Component SERVER_PLUGIN_INFO = Component.text("ℹ What is a server plugin?", INFO_COLOR)
         .append(asPlainComponents("""
             Server plugins can add new behavior to your server!
             You can find new plugins on Paper's plugin repository, Hangar.
-                        
-            <link to hangar>
-            """));
-
-    private static final Component SERVER_INITIALIZER_INFO = Component.text("ℹ What is a server initializer?", INFO_COLOR)
-        .append(asPlainComponents("""
-            Server initializers are ran before your server
-            starts and are provided by paper plugins.
+            
+            https://hangar.papermc.io/
             """));
 
     private static final Component LEGACY_PLUGIN_INFO = Component.text("ℹ What is a legacy plugin?", INFO_COLOR)
         .append(asPlainComponents("""
             A legacy plugin is a plugin that was made on
             very old unsupported versions of the game.
-                        
+            
             It is encouraged that you replace this plugin,
             as they might not work in the future and may cause
             performance issues.
             """));
 
     private static final Component LEGACY_PLUGIN_STAR = Component.text('*', TextColor.color(255, 212, 42)).hoverEvent(LEGACY_PLUGIN_INFO);
     private static final Component INFO_ICON_START = Component.text("ℹ ", INFO_COLOR);
-    private static final Component PAPER_HEADER = Component.text("Paper Plugins:", TextColor.color(2, 136, 209));
-    private static final Component BUKKIT_HEADER = Component.text("Bukkit Plugins:", TextColor.color(237, 129, 6));
+    private static final Component PAPER_HEADER = Component.text("Paper Plugins (PLUGIN_SIZE_PLACEHOLDER):", TextColor.color(2, 136, 209));
+    private static final Component BUKKIT_HEADER = Component.text("Bukkit Plugins (PLUGIN_SIZE_PLACEHOLDER):", TextColor.color(237, 129, 6));
```

Comment:
Persona... dont like this but its like the same than move to MiniMessage and apply a tag finally...

Reply:
If they aren't constants anymore, just make them into private static methods that take the int and return the component. E.g.
```java
private static Component paperHeader(int count) { return text(....) }
```